ts/screens/onboarding/EmailInsertScreen.tsx

```diff
@@ -164,11 +164,11 @@ class EmailInsertScreen extends React.PureComponent<Props, State> {
 
   private navigateToEmailReadScreen = () => {
     this.props.navigation.dispatch(StackActions.popToTop());
-    this.props.navigation.dispatch(navigateToEmailReadScreen());
+    navigateToEmailReadScreen();
```

Comment:
```suggestion
    this.props.navigateToEmailReadScreen();
```
I guess it's the same

Reply:
removed in 003f78120acd36d74bb7a9c58e86d159b24adc91